### DISCUSSION TITLE
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE

### DISCUSSION THREAD

**Discussion by mslinn** [openai/whisper#2218] [Q&A]
I am having an issue with a mismatched hash. The computer was just updated to Ubuntu 24.04. This is the command I typed to run the Whisper installer:

```
$ pip install -U git+https://github.com/openai/whisper.git --no-cache-dir
```

After downloading bazillions of packages, I get:

```
Downloading urllib3-2.2.1-py3-none-any.whl (121 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.1/121.1 kB 54.1 MB/s eta 0:00:00
Downloading nvidia_nvjitlink_cu12-12.5.40-py3-none-manylinux2014_x86_64.whl (21.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 21.3/21.3 MB 19.7 MB/s eta 0:00:00
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    unknown package:
        Expected sha256 ee53ccca76a6fc08fb9701aa95b6ceb242cdaab118c3bb152af4e579af792728
             Got        704919a474f5223b32e83f465bb84302aed65264562d83baa92168f4be7eb641
```

**Comment by montesclarosglennbenedict** [comment]
Hi! It could have been an issue with package sourcing. You could try checking each package's hash manually and reinstalling them if they do not line up. You could also just use `--no-deps` to bypass.

**Comment by mslinn** [comment]
The error appears to be in the original repo, and it was recently introduced because this same command worked a month ago.

**Reply by montesclarosglennbenedict** [reply]
Okay, after searching up the hash, the hash `ee53ccca76a6fc08fb9701aa95b6ceb242cdaab118c3bb152af4e579af792728` appears to be for the package `nvidia_cublas_cu12-12.1.3.1-py3-none-manylinux1_x86_64.whl`, which is used for PyTorch. Would it be possible for you to install this package?

**Reply by mslinn** [reply]
How?
